src/atoms/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import styled, { css } from 'styled-components'
+
+const Button = styled.button`
+  background-color: ${({ theme }) => theme.colors.black};
+  color: ${({ theme }) => theme.colors.white};
+  font-weight: 400;
+  font-size: 16px;
+  padding: 16px;
+  margin: 15px;
+  border: 0;
+  width: 100%;
+
+  hover {
+    background-color: ${({ theme }) => theme.colors.orangeLight};
+  }
+`
+
+const ProductButton = styled(Button)`
```

Comment:
`ProductButton` is too specific to be placed in the `Button.jsx`. Instead, move this `ProductButton` declaration to `Product.jsx`.